backend/src/nodes/image_adj_nodes.py

```diff
@@ -38,7 +38,7 @@ def __init__(self):
             ),
         ]
         self.outputs = [ImageOutput(image_type="Input0")]
-        self.category = IMAGE_ADJUSTMENT
+        self.category = ImageAdjustmentCategory.name
```

Comment:
Looking at this, I have another idea: Let's make it `self.category = ImageAdjustmentCategory`.

This would make `self.category` an actual category object and not just some string, and we could save the `.name` everywhere. We would just have to change the type of `self.category` and make it `"category": node_object.get_category().name` in `run.py`.

What do you think?

Reply:
Sure